probeinterface/probebunch.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+
+
+class ProbeBunch:
```

Comment:
Maybe call it "ProbeGroup"? Don't love this name currently.

Reply:
OK I have to think more about it.
Alessio did the same comments.
I was happy with Bunch.

Reply:
bunch is more peotic no ? group is too simple ...
I don't known
I am bad in english names.

Reply:
Bunch is fine too! I don't have a strong preference.